articles/quickstart/native/maui/interactive.md

```diff
@@ -105,7 +105,7 @@ using Auth0.OidcClient;
 var client = new Auth0Client(new Auth0ClientOptions
 {
     Domain = "${account.namespace}",
-    ClientId = "${account.namespace}",
+    ClientId = "{yourClientID}",
```

Comment:
```suggestion
    ClientId = ${account.clientId}
```

Reply:
Good catch, we should not use namespace.

However, we do need account.clientId as that is auto populated when reading through the quickstart while logged in.

Reply:
When not logged in, these values will be replaced with other placeholders, such as yourDomain and yourClientId.